packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -33,6 +33,8 @@ import {
   setUser,
   setUsernameAuthAttributes,
 } from '../actions';
+import { confirmSignIn, signIn } from '../defaultAuthServices';
```

Comment:
Looks like these are imported but not used at all. I do not think `defaultServices` are updated either to use `defaultAuthServices` yet.

Do we need to update `defaultServices` to use the new functions in `defaultAuthServices`?

Reply:
I'll refactor out the `defaultAuthServices`